vivisect/defconfig.py

```diff
@@ -4,6 +4,9 @@
 
 defconfig = {
     'viv':{
+        'extensions':{
+            'autocreatedir': True,
```

Comment:
i'm still of a opinion that this should be auto-save, with the option to disable autosave.

@williballenthin @mr-tz , what do you think here?  i understand your desire to have the easy ability to not create the `.viv` directory (or associated config files).  because i want to support you, and because i believe your use-case to be more programmer-friendly, is it ok if we give you a simple lever to pull when using the Viv libraries such that configuration files are not autosaved?
i would like the rest of our user-base to default to having their config items saved, but provide you with a simple way to have the environment as you want it.
i would just like the system to *default* to autosaving configuration changes.

thoughts?  concerns?

Reply:
I think related issues came up in https://github.com/vivisect/vivisect/issues/306 and when users where confused where `.viv` files (or directories) come from.
I think as long as it's easily configurable we can deal with both options. Though I'm leaning to set it to `False` here - especially if the options are not relevant to tools using vivisect as a library (maybe you can clarify the use here?).

Reply:
This definitely needs to default to False.